DSCResources/MSFT_xDnsServerForwarder/MSFT_xDnsServerForwarder.schema.mof

```diff
@@ -0,0 +1,7 @@
+[ClassVersion("1.0.0"), FriendlyName("xDnsServerForwarder")]
+class MSFT_xDnsServerForwarder :  OMI_BaseResource
+{
+    [Key] String IPAddress;
+    [Write] Boolean RemoveAll;
```

Comment:
should this be ReplaceAll?    
Additionally there is an idempotency problem here.  If you want to specify multiple and replace make sure that the ones you specify, you will be removing the other ones you specified.  Causing your configuration to constantly churn.  If your IPAddress was an array, it would fix that, but then it cannot be the key.

If this is intended to only be specified once per configuration, that still might be ok, there is a pattern to follow, see [Guidance issue 80](https://github.com/PowerShell/DscResources/issues/80)